### ISSUE TITLE
List of Google owned domains

### ISSUE THREAD

**Issue by emanruse** [Eloston/ungoogled-chromium#776]
Perhaps consider using this list (and other similar ones) in domain substitutions.

https://ipfs.io/ipfs/QmXoypizjW3WknFiJnKLwHCnL72vedxjQkDDP1mXWo6uco/wiki/List_of_Google_domains.html

**Comment by Eloston** [comment]
While we could just add all of these domains, I don't want to make the list too long and harder to read/maintain. I'd like to see how (if at all) these domains are being used in the code first.

**Comment by pipboy96** [comment]
@emanruse There are more. Getting "all" google domains is not really possible. You can theoretically block each domain that resolves to an IP with Google ASN but that's not going to work. You will just end up blocking entire Internet. I have already encountered this problem when working on HTTPS Everywhere.

**Comment by emanruse** [comment]
@Eloston

> While we could just add all of these domains, I don't want to make the list too long and harder to read/maintain. I'd like to see how (if at all) these domains are being used in the code first.

Of course. I just wanted to share a reference I found in case you want to use it as you explained.
